src/main/java/com/igorbochkor/Main.java

```diff
@@ -52,6 +46,6 @@ public static void main(String[] args) {
         carService.addDriverToCar(driverTwo, carTwo);
         System.out.println(carService.getAll());
         System.out.println(carService.getAllByDriver(driverTwo.getId()));
-        carService.delete(car.getId());
+        carService.delete(car.getId());*/
```

Comment:
can we remove the commented code? probably we can remove the Main class

Reply:
removed